app/recordtransfer/management/commands/send_test_email.py

```diff
@@ -0,0 +1,156 @@
+        """Retrieve or create a test user with the given email address.
+
+        Args:
+            email (str): The email address of the test user.
+
+        Returns:
+            User: The retrieved or newly created User instance.
+        """
+        user, _ = User.objects.get_or_create(
+            username="testuser",
+            defaults={
+                "email": email,
+                "first_name": "Test",
+                "last_name": "User",
+                "is_active": True,
+                "gets_notification_emails": True,
+                "gets_submission_email_updates": True,
+            },
+        )
+        return user
+
+    def create_test_submission(self, user: User) -> Submission:
+        """Create and return a test Submission instance for the given user.
+
+        Args:
+            user (User): The user for whom the submission is created.
+
+        Returns:
+            Submission: The created Submission instance.
+        """
+        return Submission.objects.create(
+            user=user,
+            raw_form=b"",
+            bag_name="test-bag",
+        )
+
+    def create_test_in_progress_submission(self, user: User) -> InProgressSubmission:
+        """Create and return a test InProgressSubmission instance for the given user."""
+        upload_session = UploadSession.objects.create(
+            started_at=timezone.now(),
+            status=UploadSession.SessionStatus.CREATED,
+            last_upload_interaction_time=timezone.now() - timedelta(minutes=10),
+        )
+        return InProgressSubmission.objects.create(
+            user=user,
+            title="In-Progress Test",
+            upload_session=upload_session,
+        )
+
+    def send_email(
+        self,
+        email_id: str,
+        user: User,
+        form_data: dict,
+        submission: Submission,
+        in_progress: InProgressSubmission,
+    ) -> None:
+        """Send a test email based on the provided email ID.
+
+        Args:
+            email_id (str): The ID of the email to send (email function name).
+            user (User): The user object to whom the email is related.
+            form_data (dict): Form data to be passed to the email function.
+            submission (Submission): A test submission instance.
+            in_progress (InProgressSubmission): A test in-progress submission instance.
+
+        Raises:
+            CommandError: If no handler is implemented for the given email ID.
+        """
+        func = EMAIL_FUNCTIONS[email_id]
+
+        if email_id in ("submission_creation_success", "thank_you_for_your_submission"):
+            func(form_data, submission)
+        elif email_id in ("submission_creation_failure", "your_submission_did_not_go_through"):
+            func(form_data, user)
+        elif email_id == "user_activation_email":
+            func(user)
+        elif email_id == "user_account_updated":
+            func(user, {"subject": "Your Account Was Updated"})
```

Comment:
You're missing some context here.

![Screenshot 2025-05-21 114822](https://github.com/user-attachments/assets/8b9664d7-7ad7-4ac7-9586-ce5704520283)

Try this:

```suggestion
            func(user, {
                "subject": "Account updated",
                "changed_item": "account",
                "changed_status": "updated",
                "changed_list": "Staff privileges have been added to your account.",
            })
```

See here for where this context is created in the app: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/blob/a8967e7cc82bf705f1f9bdbc410d13f58d2d5706/app/recordtransfer/admin.py#L651